doc/FormattingDecisionAssociative.md

```diff
@@ -0,0 +1,402 @@
+# Formatting Decision: Left Associative Equal and Dolon Operators
+
+We have decided to keep `=` and `::` operators left associative even while other binary operators are rewritten to be right associative.
+We realize this solution is not perfect and in this document we hope to show why this problem is too hard and we had to compromise.
+
+## Explanation
+
+Binary operators are parsed as right associative by default: `A ++ B ++ C` is parsed as `A ++ (B ++ C)`
```

Comment:
Not all binary operators. Some operators are left associative - https://erlang.org/doc/reference_manual/expressions.html#operator-precedence